executor/common_linux.h

```diff
@@ -722,6 +757,32 @@ static void initialize_devlink_pci(void)
 #endif
 #endif
 
+#if __NR_syz_80211_join_ibss || __NR_syz_80211_inject_frame || SYZ_EXECUTOR || SYZ_NET_DEVICES
+
+#define NETDEV_80211_INTITIAL_DEVICE_COUNT 2
+#define NETDEV_80211_MAC_BASE                      \
+	{                                          \
+		0x08, 0x02, 0x11, 0x00, 0x00, 0x00 \
+	}
+#define NETDEV_80211_IBSS_BSSID                    \
+	{                                          \
+		0x50, 0x50, 0x50, 0x50, 0x50, 0x50 \
+	}
+#define NETDEV_80211_DEFAULT_FREQUENCY 2412
+#define NETDEV_80211_DEFAULT_SIGNAL 0
+#define NETDEV_80211_DEFAULT_RX_RATE 1
+
+// consts from drivers/net/wireless/mac80211_hwsim.h
+#define HWSIM_CMD_REGISTER 1
+#define HWSIM_CMD_FRAME 2
+#define HWSIM_CMD_NEW_RADIO 4
+#define HWSIM_CMD_DEL_RADIO 5
```

Comment:
I think DEL and GET isn't used yet. In case it'll be used in a follow-up PR it could possibly stay, but prefer removing them.

Reply:
Removed these extra definitions.